.github/workflows/linux-build-and-test.yaml

```diff
@@ -168,6 +168,9 @@ jobs:
         make -k -C build installcheck IGNORES="${{ join(matrix.ignored_tests, ' ') }}" \
             SKIPS="${{ join(matrix.skipped_tests, ' ') }}" ${{ matrix.installcheck_args }} \
             | tee installcheck.log
+        find . -name postmaster.json
```

Comment:
Same here: I guess these three lines are only for debugging and will be removed in the final PR. Or are they used for something?